soroban-env-host/src/host.rs

```diff
@@ -1588,6 +1588,35 @@ impl CheckedEnv for Host {
         Ok(self.add_host_object(sign_bytes.1)?.into())
     }
 
+    fn bigint_from_bytes_be(&self, sign: RawVal, bytes: Object) -> Result<Object, Self::Error> {
+        let s = self.bigint_sign_from_rawval(sign)?;
+        let res = self.visit_obj(bytes, |b: &Vec<u8>| {
+            MeteredBigInt::from_bytes_be(self.0.budget.clone(), s, b)
+        })?;
+        Ok(self.add_host_object(res)?.into())
+    }
+
+    fn bigint_from_radix_be(
+        &self,
+        sign: RawVal,
+        buf: Object,
+        radix: RawVal,
+    ) -> Result<Object, Self::Error> {
+        let s = self.bigint_sign_from_rawval(sign)?;
+        let r = self.bigint_radix_from_rawval(radix)?;
+        let res = self.visit_obj(buf, |b: &Vec<u8>| {
+            if r != 256 && b.iter().any(|&b| b >= r as u8) {
```

Comment:
What's the significant of r != 256? Does this mean the radix has to be 256?

Reply:
Oh I see, this check only occurs when the radix is < 256. There's no check here that `r` isn't > 256.

Reply:
This is just to skip the comparison for slight performance improvement. Since in `bigint_radix_from_rawval` we made sure that `r` is <= 256.

Reply:
Oh, I'm wrong, the check happens inside `bigint_radix_from_rawval`.